core/src/main/java/org/apache/accumulo/core/util/Wait.java

```diff
@@ -51,6 +56,14 @@ public static int getTimeoutFactor(ToIntFunction<NumberFormatException> onError)
 
   }
 
+  public static void sleep(long millis) {
+    try {
+      Thread.sleep(millis);
+    } catch (InterruptedException e) {
+      log.error("{}", e.getMessage(), e);
```

Comment:
Should `Thread.currentThread().interrupt();` go here?

Reply:
Probably, but it was not in the original code - and I don't think anyone is actually checking the flag status anyway.  It would be better if we did not use this method and examined interrupt handling in general, but that is beyond the scope of this PR.